aiogram/dispatcher/dispatcher.py

```diff
@@ -450,7 +451,7 @@ async def start_polling(
         polling_timeout: int = 10,
         handle_as_tasks: bool = True,
         backoff_config: BackoffConfig = DEFAULT_BACKOFF_CONFIG,
-        allowed_updates: Optional[List[str]] = None,
+        allowed_updates: Optional[List[str]] = UNSET_TYPE,
```

Comment:
Is the type hint correct?

Reply:
Seems like it replaced `UNSET`, which is not available in the current version.